File: contests/codeforces/contest404/A.hs
import Control.Monad

getFaceCount name
    | name == "Tetrahedron" = 4
    | name == "Cube" = 6
    | name == "Octahedron" = 8
    | name == "Dodecahedron" = 12
    | name == "Icosahedron" = 20

main :: IO()
main = do
    w <- getLine
    s <- forM [1..(read w :: Int)] $ \i -> getLine
    print $ sum (map getFaceCount s)

File: contests/codeforces/contest404/A.py
faces = {
        "Tetrahedron" : 4,
        "Cube" : 6,
        "Octahedron" : 8,
        "Dodecahedron" : 12,
        "Icosahedron" : 20
        }

if __name__ == "__main__":
    n = int(raw_input())
    face_count = 0
    for i in xrange(n):
        face_count += faces[raw_input()]
    print face_count
